src/main/pg/gps.c

```diff
@@ -29,7 +29,7 @@
 
 #include "gps.h"
 
-PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 4);
+PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 3);
```

Comment:
The variables for the GPS PG were not changed in PR #13240, so the PG did not increasing from 3 to 4 in that PR.
This PR is basically a reversion of 13240, and it seems correct to keep the PG at 3, especially since nothing changed.
However, if it is best to keep it at 4 please let me know.